dashboards/grafana-dashboard-assisted-installer-ocm-operations.configmap.yaml

```diff
@@ -148,10 +148,10 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(service_assisted_installer_operation_duration_miliseconds_bucket{le!=\"+Inf\"}[5m])) by (operation)",
+              "expr": "sum(rate(service_assisted_installer_operation_duration_miliseconds_count[5m])) by (operation)",
```

Comment:
for some reason it did not submit my comment, so rewriting it:
I propose `$__rate_interval` instead of `5m` see details here: https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/

Reply:
Changed it.